manifests/init.pp

```diff
@@ -34,13 +34,13 @@
 #                               type:Optional[Stdlib::Absolutepath]
 #
 # $forward::                    The forward option
-#                               type:Optional[String]
+#                               type:Optional[Enum['only', 'first']]
```

Comment:
As someone still a bit new to types: is there a difference between quoted and unquoted values here?